.prettierignore

```diff
@@ -3,4 +3,3 @@ staking/lib
 wasm
 staking/app/keypairs
 docs/
-frontend/
```

Comment:
This is preventing the pre-commit to check frontend

Reply:
There is a separate github action for prettier specifically now so pre-commit shouldnt be needed.

The pre-commit prettier plugin is long deprecated and the version of prettier that it ships is very out of date and actually is not compatible with modern prettier so IMO we should not use it any more and should use prettier directly instead.